components/fallback/PropertyProposals.vue

```diff
@@ -149,6 +153,15 @@ export default {
       return _get(proposal, 'answers.messages', []).length
     }
   },
+  computed: {
+    hasProposals () {
+      return (
+        this._get(this.newObjectProposals, 'proposals.length') ||
+        this._get(this.changedClassProposals, 'proposals.length') ||
+        this._get(this.changedPropertyProposals, 'proposals.length')
```

Comment:
Could you please extract each of these conditions as their own computed property and then use these in the template? It would reduce logic duplication a little bit.